doc/Learn/Maui/MauiOverview.md

```diff
@@ -16,9 +16,12 @@ It's recommended that instead of hosting individual .NET MAUI controls in a `Mau
 
 ## Get Started - New Uno Application
 
+> [!NOTE]
+> Make sure to setup your environment first by [following our instructions](xref:Uno.GetStarted.vs2022).
+
 For new applications created using the Uno Platform Template Wizard for Visual Studio, .NET MAUI Embedding can be selected as a feature when creating the application.
 
-1. Start by selecting either the Blank or Default template, and clicking the Customize button (since neither of the predefined configurations include the .NET MAUI Embedding by default).
+1. Start by selecting either the **Blank** or **Recommended** preset template.
 
     ![Select startup type](Assets/GettingStarted01-StartupType.png)
 
```

Comment:
Need to confirm but I don't think the "Select the Platforms section and make sure only....." is relevant any more

Reply:
Yes, you are right. The **.NET MAUI Embedding** feature is still enabled even with all the platforms selected. Can we remove this step? I would suggest we add a note stating the supported platforms, but it turns out that the first paragraph of this page already mentions it.

Reply:
yeh I think we can remove this step